src/codegen/helpers.rs

```diff
@@ -143,6 +143,25 @@ pub mod ast_ty {
     use proc_macro2::{self, TokenStream};
     use std::str::FromStr;
 
+    pub fn c_void(ctx: &BindgenContext) -> TokenStream {
+        // ctypes_prefix takes precedence
+        match ctx.options().ctypes_prefix {
+            Some(ref prefix) => {
+                let prefix = TokenStream::from_str(prefix.as_str()).unwrap();
+                quote! {
+                    #prefix::c_void
+                }
+            }
+            None => {
+                if ctx.options().use_core {
```

Comment:
Given this is not in rust 1.0, we should probably add a member to the rust features (https://github.com/rust-lang/rust-bindgen/blob/master/src/features.rs), and check for that here. WDYT?

Reply:
Sounds good to me. I've update accordingly.